fileonfeature01.txt

```diff
@@ -1,2 +1,3 @@
 this is my file on feature branch
-added some lines to edit the file
\ No newline at end of file
+added some lines to edit the file
+new line on new branch
```

Comment:
This is not the way how we add new lines to our txt files! It should start: "Added for ex."